### DISCUSSION TITLE
Disable Vanilla Crafting Recipes

### DISCUSSION THREAD

**Discussion by Saymoan** [Zrips/CMI#3501] [Feature Requests]
**Description of issue or feature request:**
Ability to disable Vanilla crafting recipes, in a GUI similar to disabling Vanilla enchantments.

**Comment by CloudeLecaw** [comment]
So you wanne disable vanilla craftings?

for older servers with IDs
https://dev.bukkit.org/projects/itemblock-disabler2

or this one

r never server with itemnames
https://www.spigotmc.org/resources/limitcrafting-1-11-updated-disable-crafting-recipes.12962/

or may  look for this

https://www.spigotmc.org/resources/recipecontrol.8152/

there is also a premium plugin:
https://www.spigotmc.org/resources/now-multilingual-%E2%9C%AE-recipe-control-%E2%9C%AE-1-8-8-1-14-x-%E2%9C%AE-supports-new-1-14-recipes.67216/


Even if there are a loot of plugins on the market it sounds like a nice idea for a further Request.  Im just not so sure if creating a gui for this is the best idea, maybe a configuratiolist by command where craftings can be added/listed/removed by the name would be a smarter solution? 

have a nice day

**Comment by broken1arrow** [comment]
> 
> 
> So you wanne disable vanilla craftings?
> 
> for older servers with IDs
> https://dev.bukkit.org/projects/itemblock-disabler2
> 
> or this one
> 
> r never server with itemnames
> https://www.spigotmc.org/resources/limitcrafting-1-11-updated-disable-crafting-recipes.12962/
> 
> or may look for this
> 
> https://www.spigotmc.org/resources/recipecontrol.8152/
> 
> there is also a premium plugin:
> https://www.spigotmc.org/resources/now-multilingual-%E2%9C%AE-recipe-control-%E2%9C%AE-1-8-8-1-14-x-%E2%9C%AE-supports-new-1-14-recipes.67216/
> 
> Even if there are a loot of plugins on the market it sounds like a nice idea for a further Request. Im just not so sure if creating a gui for this is the best idea, maybe a configuratiolist by command where craftings can be added/listed/removed by the name would be a smarter solution?
> 
> have a nice day

Some of this not work on 1.13+, is only the premium is for 1.13+. I have look on other free plugins and they not work properly (some have more stuff, what you need). 

I try one some add recipe too the book, but middle click not work, is for you can edit or delete recipe (he has not fix it, even if i report it). It had permissions (if you want it), for recipe shall be made.

Other was over complicate and not use vanilla recipe book.

Easy plugin, some you can add custom recipe with or with out NBT and not disappear when you restart server.

The premium, is sad you cant try it. I want plugin, some has NBT suport and can set permissions (if you want it).

**Comment by CloudeLecaw** [comment]
you look for the free once but you have CMI? ;) *joking*
What i can recommend is this one 
[Recipe Control](https://www.spigotmc.org/resources/recipe-control-%E2%9C%AE-1-15-1-16-%E2%9C%AE-edit-and-create-recipes.67216/)

yes i know its premium but its good and not too expensive... we used this on our server to disable some recipes and its still up to date even if you have 1.16.1 for 1.13.2 you need the version lower then 3.x
i dont know if the newer one work with 1.13.2 because they where made for 1.15.x and 1.16.x

**Comment by broken1arrow** [comment]
> 
> 
> you look for the free once but you have CMI? ;) _joking_
> What i can recommend is this one
> [Recipe Control](https://www.spigotmc.org/resources/recipe-control-%E2%9C%AE-1-15-1-16-%E2%9C%AE-edit-and-create-recipes.67216/)
> 
> yes i know its premium but its good and not too expensive... we used this on our server to disable some recipes and its still up to date even if you have 1.16.1 for 1.13.2 you need the version lower then 3.x
> i dont know if the newer one work with 1.13.2 because they where made for 1.15.x and 1.16.x

I want to try before buy :) I want option to set permissions on recipe (players can buy recipe) or can get a recipe for free (you cant do it on CMI even if is premium plugin) :) and recipe stick in the green book and it do that?

We use 1.15+ now and soon 1.16+ so no problems :)

 Don't need block recipes.

**Comment by CloudeLecaw** [comment]
> **Description of issue or feature request:**
> Ability to disable Vanilla crafting recipes, in a GUI similar to disabling Vanilla enchantments.

this is was the Header said and want^^
i thought u may need the same. 

> >  want option to set permissions on recipe (players can buy recipe) or can get a recipe for free (you cant do it on CMI even if is premium plugin) :) and recipe stick in the green book and it do that?
> 
> We use 1.15+ now and soon 1.16+ so no problems :)
> 
> Don't need block recipes.

Buying recipes -nice idea : - D
maybe in combinated with a Onetime permission so you need to buy the recipe aggain and aggain^^

**Comment by broken1arrow** [comment]
> 
> 
> > **Description of issue or feature request:**
> > Ability to disable Vanilla crafting recipes, in a GUI similar to disabling Vanilla enchantments.
> 
> this is was the Header said and want^^
> i thought u may need the same.
> 
> > > want option to set permissions on recipe (players can buy recipe) or can get a recipe for free (you cant do it on CMI even if is premium plugin) :) and recipe stick in the green book and it do that?
> > 
> > 
> > We use 1.15+ now and soon 1.16+ so no problems :)
> > Don't need block recipes.
> 
> Buying recipes -nice idea : - D
> maybe in combinated with a Onetime permission so you need to buy the recipe aggain and aggain^^

Yes that was a nice evil idea XD but no, more like newbe, not craft lots of stuff take server resources. We have made so you can craft hoppers and chests from other plugins (hopper and chest plugins we get linking and can filter items and chest like crafting and sell chest, best is chest you can storage unlimited of one item (you can limit it if you want)). 

Storage chest has sadly one backside (if you shift in wrong item (if you already have one item type in the chest) and right item on top, the first disappear). So is best lock out new on the server (they are like use it wrong and complain on staff, to fix back items) :) 

But is not good tog give to new on server access too all recipe (if they don't have basic understanding already). Better they play and learn over time how it work (recipes liked to plugins at least shall not be easy to get):)

Especially sell chest, is powerful (can sell items auto directly from a farm). The idea of ​​this chest is you buy one chest and get a recipe so you can make your own (but perhaps bad idea :)). Some CMI are now, can we not use recipe if we sell this items in a shop (is not to hard to find recipe). We can lock down /recipe (but can be handy for players some know this command)

I don' t see other thread for suggestion on recipe, so i took this (some are not to far away). 

But you cant set permissions on this plugin?

**Comment by CloudeLecaw** [comment]
Permissions on cmi?

**Comment by broken1arrow** [comment]
> 
> 
> Permissions on cmi?

no in Recipe Control (also for recipe) (or have cmi recipe permissions ;))?
